group-generators/generators/sismo-hackathons-participants/index.ts

```diff
@@ -0,0 +1,56 @@
+
+import { Tags, ValueType, GroupWithData } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  
+  generationFrequency: GenerationFrequency.Daily,
+  
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+  
+    const builders = {
```

Comment:
It seems to be very few members

Reply:
You are right
I saw with Sasha there was a slight misunderstanding, I'll fix this, thanks!